ledger-tool/src/bigtable.rs

```diff
@@ -584,6 +830,124 @@ impl BigTableSubCommand for App<'_, '_> {
                                 .takes_value(false)
                                 .help("Display the full transactions"),
                         ),
+                )
+                .subcommand(
+                    SubCommand::with_name("copy")
+                        .about("copy blocks from a bigtable to another bigtable")
+                        .arg(
+                            Arg::with_name("source_credential_path")
+                                .long("source-credential-path")
+                                .value_name("SOURCE_CREDENTIAL_PATH")
+                                .takes_value(true)
+                                .help(
+                                    "Source bigtable creds path. (could be readonly)",
+                                ),
+                        )
+                        .arg(
+                            Arg::with_name("source_endpoint")
+                                .long("source-endpoint")
+                                .value_name("SOURCE_ENDPOINT")
+                                .takes_value(true)
+                                .help(
+                                    "Source bigtable endpoint.",
+                                ),
+                        )
+                        .arg(
+                            Arg::with_name("source_is_emulator")
+                            .long("source-is-emulator")
```

Comment:
```suggestion
                            .long("emulator-source")
```
Wdyt?

Reply:
`emulated-source` ?

Reply:
I combined `source_is_emulator` and `endpoint` into `emulated_source`. (same as destination part) https://github.com/solana-labs/solana/pull/28122/commits/79fe770ccab4d09dec014ef3b00bbdb17a12ebda